### PR TITLE
[ninateka] Add new extractor

### PR DESCRIPTION
**Pull request by Tithen-Firion**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [ ] Improvement
- [x] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Add a new extractor for [Ninateka](https://ninateka.pl/)

### CODE REVIEW

**Comment by angularcircle** [comment]
Tried on some _Ninateka_ videos - works well. Thanks!
This pull request will also solve issue #16023

**Comment by haridutt12** [comment]
It looks like this PR is not following the [copyright infringement](https://github.com/ytdl-org/youtube-dl/blob/master/README.md#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free) of this repository. as I checked on [ninateka]https://ninateka.pl/filmy/muzykal) website, they have All Rights Reserved.

**Comment by Messere** [comment]
> It looks like this PR is not following the [copyright infringement](https://github.com/ytdl-org/youtube-dl/blob/master/README.md#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free) of this repository.

Ninateka is run under the aegis of Polish Ministry of Culture. It certainly neither host videos that infringe copyright, nor links to such content. All you can find there is legitimate.

**Comment by selfisekai** [comment]
Ninateka is being run by National Audiovisual Institute (Narodowy Instytut Audiowizualny), which got merged with National Film Library (Filmoteka Narodowa) - now called [Filmoteka Narodowa – Instytut Audiowizualny](https://pl.wikipedia.org/wiki/Filmoteka_Narodowa_–_Instytut_Audiowizualny).
this state-owned institution receives money from Ministry of Culture and National Heritage for its projects. here is a list of them: https://fina.bip.gov.pl/projekty-dofinansowane-ze-srodkow-ministra-kultury-i-dziedzictwa-narodowego/projekty-dofinansowywane-z-mkidn.html, which includes _payment of fees for rights and licenses to the contents of the NInA website_ (_realizacja opłat za prawa i  licencje do zamieszczanych na stronach  internetowych NInA treści_).

TL;DR state-owned website, pays to make content available legally for free

**Comment by selfisekai** [comment]
Ninateka replaced their frontend with a new one, also changing the URL scheme.

- video: https://ninateka.pl/vod/dla-dzieci/dziwny-swiat-kota-filemona-wielkie-pranie/
- audio: https://ninateka.pl/vod/muzyka/drony-drony-fisz-emade-tworzywo-11-12/

**Comment by selfisekai** [comment]
Turns out that all medias use Widevine/Playready/Fairplay protection now, the PR can be closed.
